src/main/java/org/dataone/cn/indexer/IndexWorker.java

```diff
@@ -311,179 +312,162 @@ protected void initExecutorService() {
-                    + pid.getValue() + " since " + e.getMessage());
-        } catch (XPathExpressionException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (UnsupportedType e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (SAXException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (ParserConfigurationException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (SolrServerException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (MarshallingException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (EncoderException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (InterruptedException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (IOException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (InvalidRequest e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage());
-        } catch (InstantiationException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
-        } catch (IllegalAccessException e) {
-            logger.error("IndexWorker.indexOjbect - cannot index the task for identifier "
-                    + pid.getValue() + " since " + e.getMessage(), e);
+                            + pid.getValue() + " , the index type: " + indexType
+                            + ", the file path (null means not to have): " + finalFilePath
+                            + ", the priority: " + priority + " and the time taking is "
+                            + (end - start) + " milliseconds");
+
+        } catch (InvalidToken | NotAuthorized | NotImplemented | NotFound | InvalidRequest |
```

Comment:
in metadig I separate these into groups and handle them differently. Something like a `ServiceFailure` could/should be just be retried since its likely a blip on the node, but `InvalidToken` indicates the token is likely expired (or wasn't configured correctly) which requires intervention.